dynamic.js

```diff
@@ -106,6 +106,7 @@ module.exports = function (fastify, opts, next) {
       swaggerObject.externalDocs = externalDocs
     }
 
+    /* istanbul ignore next */
```

Comment:
You should add a test where `ref` is falsy.

Reply:
@mcollina 
The `ref` is inside `dynamic.js` set on [line 13](https://github.com/fastify/fastify-swagger/blob/master/dynamic.js#L13) and has `undefined` value. 

I believe that was done on purpose. If we initialize it with some values, it may affect the existing logic. Therefore, we cannot change its value. 

So, what do you suggest?

Reply:
I mean there should be a test where we are calling the `app.swagger()` function twice.